addons/l10n_in_tcs_tds/data/account.account.template.csv

```diff
@@ -0,0 +1,3 @@
+"id","name","code","account_type","chart_template_id/id","tag_ids/id","reconcile"
+"p11244","TDS Deducted","11244","liability_current","l10n_in.indian_chart_template_standard","","False"
```

Comment:
As we do everything in a separate module, we need some kind of post_init_hook still to allow the new taxes/accounts to be loaded if the CoA got installed in the current company with the installation of the module.  (or if we want to have them in the existing companies after installing this module)

Reply:
@jco-odoo @jva-odoo , I have added post_init hook.